content/TrueCommand/TCGettingStarted/Install/VMDeploy.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "VM Deployment"
+weight: 40
```

Comment:
Ensure article is first in the list of "Installing" subtopics
```suggestion
weight: 10
```

Reply:
Alright. I just chose 40 because the others were 50 and 60. Fixed!